[CloudPC] Add powerState in CloudPC entity (#20511)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/cloudpc.md

|onPremisesConnectionName|String|The Azure network connection that is applied during the provisioning of Cloud PCs.|
|osVersion|[cloudPcOperatingSystem](../resources/cloudpcorganizationsettings.md#cloudpcoperatingsystem-values)|The version of the operating system (OS) to provision on Cloud PCs. Possible values are: `windows10`, `windows11`, and `unknownFutureValue`.|
|partnerAgentInstallResults|[cloudPcPartnerAgentInstallResult](../resources/cloudpcpartneragentinstallresult.md) collection|The results of every partner agent's installation status on Cloud PC.|

> |powerState|[cloudPcPowerState](#cloudpcpowerstate-values)|The Power State of the Cloud PC. Possible values include running, poweredOff. This is only supported for Cloud PCs with the provisioningType set to 'shared'.|

Reviewer: This is only supported for shift work CloudPCs.

|onPremisesConnectionName|String|The Azure network connection that is applied during the provisioning of Cloud PCs.|
|osVersion|[cloudPcOperatingSystem](../resources/cloudpcorganizationsettings.md#cloudpcoperatingsystem-values)|The version of the operating system (OS) to provision on Cloud PCs. Possible values are: `windows10`, `windows11`, and `unknownFutureValue`.|
|partnerAgentInstallResults|[cloudPcPartnerAgentInstallResult](../resources/cloudpcpartneragentinstallresult.md) collection|The results of every partner agent's installation status on Cloud PC.|

> |powerState|[cloudPcPowerStatus](#cloudpcpowerstatus-values)|The Power State of the Cloud PC. Possible values include `running`, `poweredOff`, `deallocated`, `hibernated`, `unknown`. This is only supported for shift work CloudPCs.|

Reviewer 2: ```suggestion
|powerState|[cloudPcPowerStatus](#cloudpcpowerstatus-values)|The power state of a Cloud PC. The possible values are: `running`, `poweredOff`, `deallocated`, `hibernated`, `unknown`. This property only supports shift work CloudPCs.|
```

|restoring|The Cloud PC is restoring.|
|unknownFutureValue|Evolvable enumeration sentinel value. Do not use.|

### cloudPcPowerStatus values

|Member|Description|
|:---|:---|
|running|The Cloud PC status is running.|
|poweredOff|The Cloud PC status is power off.|
|deallocated|The Cloud PC status is deallocated.|

> |hibernated|The Cloud PCstatus is hibernated.|

Reviewer 2: The members "deallocated" and "hibernated" are not mentioned in the CSDL or the api.md files. Please confirm if these are valid members.
